### DISCUSSION TITLE
[Hygiene] Enable Squash and Merge for Incoming PRs

### DISCUSSION THREAD

**Discussion by ankit-agarwal-skydio** [mvukov/rules_ros2#28] [Ideas]
Hey! Just wanted to start a quick discussion around enabling squash and merge for incoming PRs. Makes it a lot easier to search the commit history :)

**Comment by mvukov** [comment]
Hello! Sounds like a good idea. I took a look and squashing should work -- it's enabled. Please let me know if it doesn't, I'll look further then.
